tracerite/style.css

```diff
@@ -43,56 +57,69 @@
   padding-right: 1ch;
   width: 4ch;
 }
+
 .tracerite .traceback-details pre:hover .codeline::before {
   opacity: 1.0;
 }
+
 .tracerite .traceback-details mark {
   background: none;
 }
+
 .tracerite .traceback-details mark span {
-  background: #ff0;
+  background: var(--tracerite-highlight);
 }
+
 .tracerite .traceback-details mark::after {
   display: inline-block;
   content: attr(data-symbol);
   margin: -1ch;
   transform: translate(2em, 0) scale(1.8);
-  color: #ff0;
+  color: var(--tracerite-highlight);
   -webkit-text-stroke: .05em black;
 }
+
 .tracerite .excmessage {
   max-height: 12em;
   overflow: auto;
 }
-.tracerite .exctype {color: gray}
+
+.tracerite .exctype { color: var(--tracerite-exception) }
+
 .tracerite .traceback-labels {
   display: flex;
   align-items: center;
   border-bottom: 3px solid var(--tracerite-tab);
   margin-top: 0.3em;
 }
+
 .tracerite .traceback-labels button {
-  background: var(--tracerite-tab); color: black;
+  background: var(--tracerite-tab);
+  color: black;
   border: 0;
   border-radius: .5em .5em 0 0;
   flex-shrink: 1;
   line-height: 1.0;
   padding: .5em;
   margin-right: .2em;
 }
-.tracerite .traceback-labels button:hover { background: #ddd }
+
+.tracerite .traceback-labels button:hover { background: var(--tracerite-label) }
```

Comment:
This should be `--tracerite-tab-hover`

Reply:
feels very specific, but OK

Reply:
The --tracerite-label is the text color on those tooltips, it would appear. Two different things.